snap/local/helper-go/configure.go

```diff
@@ -509,48 +509,35 @@ func checkSecurityConfig(services []string) ([]string, error) {
 		// default behavior
 		services = append(services, getProxyServices()...)
 	default:
-		err = fmt.Errorf("edgexfoundry:configure: invalid setting for security-proxy: %s", status)
+		err = fmt.Errorf("invalid setting for security-proxy: %s", status)
 		return nil, err
 	}
 	return services, nil
 }
 
 func configure() {
+	log.SetComponentName("configure")
+
 	// process the EdgeX >=2.2 app options
-	processAppOptions()
+	if err := processAppOptions(); err != nil {
+		log.Fatalf("error processing app options: %v", err)
+	}
 
-	var debug = false
 	var err error
 	var startServices []string
 
-	status, err := cli.Config("debug")
-	if err != nil {
-		fmt.Println(fmt.Sprintf("edgexfoundry:configure: can't read value of 'debug': %v", err))
-		os.Exit(1)
-	}
-	if status == "true" {
-		debug = true
-	}
-
-	if err = hooks.Init(debug, "edgexfoundry"); err != nil {
-		fmt.Println(fmt.Sprintf("edgexfoundry:configure: initialization failure: %v", err))
-		os.Exit(1)
-	}
-	hooks.Info("edgexfoundry:configure: started")
+	log.Info("Started")
```

Comment:
I have addressed this and your next comment in my upcoming refactoring, which will be added once this PR is merged. See https://github.com/farshidtz/edgex-go/blob/cbd7714b276f42495f659f666f7270833e296979/snap/local/helper-go/configure.go#L544-L551

If you agree, I'd like to keep it that way to avoid merge conflicts.